docs/tool/js/views/filter-view.js

```diff
@@ -191,7 +220,7 @@ var filterView = {
         }
 
         this.isTouched = function(){
-            return slider.noUiSlider.get()[0] === c.min && slider.noUiSlider.get()[1] === c.max;
+            return slider.noUiSlider.get()[0] !== c.min && slider.noUiSlider.get()[1] !== c.max;
```

Comment:
One bug I found: when you click the 'clear filters' pillbox, the function to recalculate the filteredData list is run like 25 times, which causes some serious responsiveness issues during clearing. I haven't fully investigated but I suspect that this change is the source - remember the 'isTouched' was originally added to make sure that the clear filter button didn't trigger too often. 

I don't see why this function was changed to `!==`, though technically now I think it returns the proper 'true' and 'false state per its name (isTouched = true if either the min or max is altered). If changing this, you should also change the code that uses the isTouched when deciding whether to recalculate the filters. 

Future note: the 'forcePublish' function and refactor to use the 'updateFilters' event subscription is a better long term solution to this issue, and is part of the URL encoding refactor described in #387. At that point `isTouched` might no longer be needed.